frontend/app.vue

```diff
@@ -1,8 +1,10 @@
 <template>
-  <div h-full>
-    <NuxtLayout>
-      <NuxtPage />
-    </NuxtLayout>
+  <div class="bg-[url(public/background.png)]" h-full bg-cover bg-no-repeat>
```

Comment:
I don't think that we want to make all sites have space background. create layout for it

Reply:
what do you mean?

Reply:
now I think about it its not covered by this issue, so I will just delete this from app.vue